sycl/source/detail/scheduler/commands.hpp

```diff
@@ -364,7 +364,7 @@ class ReleaseCommand : public Command {
 class AllocaCommandBase : public Command {
 public:
   AllocaCommandBase(CommandType Type, QueueImplPtr Queue, Requirement Req,
-                    AllocaCommandBase *LinkedAllocaCmd);
+                    AllocaCommandBase *LinkedAllocaCmd, bool IsConst = false);
```

Comment:
Is the default value for `IsConst` necessary? Seems like `LinkedAllocaCmd` doesn't have it here, likely because this is the base so it expects its descendants to actively choose or pass on the information.

Reply:
I cant think of a case where default value would be necessary :). Removing it to keep it more consistent with surrounding code.